Stable/cm 11.0

willneo opened the pull request.

rmcc committed:
Update manifest for cm-11

Change-Id: Ie49fb11669396bc1594e55824cae8d36ce439596

rmcc committed:
Replace the sdk repository reference with a hard sha reference

The tag apparently has different meanings/contents in different
google mirrors for this particular repo... FUN!

Change-Id: I5f9335af4da7114d83457ca041026f3be99c32ce

rmcc committed:
Switch to our own fork of ART

Change-Id: I85ca84303ce53183892911aadae921800a1415d5

rmcc committed:
Underp noto-fonts, too

Same problem as prebuilts/sdk

Change-Id: Ic47cd00acd9cb15c1b4de81aa813e7a0381045ef

rmcc committed:
4.4_r1 -> 4.4_r1.2

Change-Id: Ide5eb79266c984cea0feeee5ddd46161c56624b9

rmcc committed:
Switch to our fork of chromium_org

Change-Id: I80e11eba8482bee1c24d013c546d9b3ef2e90bc1

hyperb1iss committed:
Add BluetoothExt to cm11

Change-Id: I0cfb9ab368700a416f7275c124f444e02ed0715a

koush committed:
readd VoicePlus

Change-Id: I3a07542ff8d377044058651c0244052f43708141

hyperb1iss committed:
Add external/connectivity for CNE

Change-Id: Ia2ad2a53c89c60ed9593abb178d8ab547ac04344

rmcc committed:
Use the stabilization branch as default here...

Change-Id: I9feefb361a9fbbad2156a0bf8e607b6f2bf7913b

rmcc committed:
4.4_r1.2 -> 4.4.1_r1

Change-Id: I946441c55ae81dcce53cbcc8df21e47ebd08f686

koush committed:
Add vendor/cyngn

Change-Id: I6292ab1c1ae15985f3694232ca41d1ab178f3969

koush committed:
Merge "Add vendor/cyngn" into cm-11.0

nebkat committed:
manifest: Trebuchet

Change-Id: I387b177bd86d836efc15578fe6497cc20c18cf48

ctso committed:
Add WhisperPush

Change-Id: I147c9375616de27ffd4a603699917155547f86e9

rmcc committed:
4.4.1_r1 -> 4.4.2_r1

Change-Id: Ibb2c771339e19d53963eff10de9bc530cde2a271

hyperb1iss committed:
Remove qrngd

 * This is built into init now.

Change-Id: I0f6cde9e4da9d0f8a97076e32d6168b1e2c14595

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!

On Sun, 05 Jan 2014 17:58:09 -0800 you wrote:

> You can merge this Pull Request by running:
> 
>   git pull https://github.com/CyanogenMod/android stable/cm-11.0
> 
> Or you can view, comment on it, or merge it online at:
> 
>   https://github.com/CyanogenMod/android/pull/29
> 
> -- Commit Summary --
> - Update manifest for cm-11
> - Replace the sdk repository reference with a hard sha reference
> - Switch to our own fork of ART
> - Underp noto-fonts, too
> - 4.4_r1 -&gt; 4.4_r1.2
> - Switch to our fork of chromium_org
> - Add BluetoothExt to cm11
> - readd VoicePlus
> - Add external/connectivity for CNE
> - Use the stabilization branch as default here...
> - 4.4_r1.2 -&gt; 4.4.1_r1
> - Add vendor/cyngn
> - Merge &quot;Add vendor/cyngn&quot; into cm-11.0
> - Add WhisperPush
> - 4.4.1_r1 -&gt; 4.4.2_r1
> - Merge remote-tracking branch &#39;github/cm-11.0&#39; into HEAD
> 
> -- File Changes --
> 
> ```
> M README.mkdn (4)
> D cm-10.2-M1.xml (426)
> M default.xml (204)
> ```
> 
> -- Patch Links --
> 
> https://github.com/CyanogenMod/android/pull/29.patch
> https://github.com/CyanogenMod/android/pull/29.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/CyanogenMod/android/pull/29

thoemy committed:
Add wpa_supplicant_8_ti

Change-Id: Ia4a7318c5baa33bb15a0002a6b4ef9b4236ff1e0

Commit:
Revert "manifest: Trebuchet"

This reverts commit 9a47f558c7b1a9efe80b61f1edd512fe6e6d96f8.

The needs to be rebased without the package rename before we bring it in.

Change-Id: Ib8f79adef1cf3b04d54b30dfccd681ede7c44dab

hyperb1iss committed:
Revert "Remove qrngd"

 * Bring this back. It improves boot time on several devices. The
   built-in seeder is oneshot.

This reverts commit 89b023b691148d299bed54b5c4807cfb72e61a4f.

Change-Id: Ic9e272b7a5d797d06e9a3109328ef6aa0a13b172

rmcc committed:
Revert "Revert "manifest: Trebuchet""

Blockers are all out of the way, lets do this thing.
This reverts commit fd42a60e05f2bee1a29f78a5ab48d1a6b79eadaf.

Change-Id: Icc050376c32af0cefe7a71b3c3a58da5f9bb518d

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

mikeNG committed:
manifest: Track our own prebuilts/misc

Change-Id: I82366a40841b2c535ef6dc68e1c1cd3b914ded27

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

hyperb1iss committed:
Add telephony-msim to the build

Change-Id: I6e18f97ce162476a8d3c339997005d1515361306

slickrick committed:
manifest: Dont track QuickSearchBox

We removed it here I18548dd0b61cc8dedf6570562d073d0973120cba so no point tracking it

Change-Id: I3b7d8d498246ba4882af4ee9a13f03b66de2e89a

rmcc committed:
4.4.2_r1 -> 4.4.2_r2

Change-Id: I88d21c3cfd138dd8d5b23c23b5d294c138663850

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

mikeNG committed:
manifest: Don't track Launcher3

Change-Id: I34fac74b2b44c52828f65fb9232e438aae533dc1

hyperb1iss committed:
Add protobuf-c project to the build

 * Needed for build of RIL on some devices.

Change-Id: I49f1db8bbc9df522679fd887d05d8ad0dc0ca702

hyperb1iss committed:
Add CURL and BSON to the build.

 * Needed for CNE and other stuff

Change-Id: Iff259c2abbf2b25a35e95708d66ba015d3b52901

rmcc committed:
Track our own copy of chromium.org's openssl

Change-Id: I3e82075981caee1b590067a8706d037a6c3dd4b1

rmcc committed:
Merge "manifest: Dont track QuickSearchBox" into cm-11.0

rmcc committed:
Merge "manifest: Don't track Launcher3" into cm-11.0

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

tdmcyngn committed:
manifest: track libtar

Change-Id: Ia1d490d3ec4eb99df05553806c38bd8b33d15f68

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

euclidgeo committed:
New Theme Chooser UI

Change-Id: I5bf43f3e14c391af3263db8f3bfb8afa39be3928

romanbb committed:
manifest: track android-visualizer

Change-Id: I52f8457c074c633b322d50bd8d7e04280b5547b6
Signed-off-by: Roman Birg <roman@cyngn.com>

hyperb1iss committed:
Add FFMPEG and stagefright-plugins to the project

Change-Id: Iec58ed53f953304d8b78fd7561548fa771b33349

euclidgeo committed:
Merge "New Theme Chooser UI" into cm-11.0

tdmcyngn committed:
manifest: Add simplerecovery (bootable/recovery-cm)

Change-Id: I3ee0149fa06d792f5c0b00c4b02f61183dab9808

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

hyperb1iss committed:
Add ebtables

Change-Id: I94923aacb95e0a5dc22f95730e0fc6754f6c6ab7

Commit:
Merge remote-tracking branch 'github/cm-11.0' into HEAD

rmcc committed:
4.4.2_r2 -> 4.4.3_r1

Change-Id: Ic14bbfc61e55bc8d26c0bbbe97a18e00a088326e

rmcc committed:
4.4.3_r1 -> 4.4.3_r1.1

Change-Id: I018a4fbbae6501705230624df7d2a3ba3c320b03

ctso committed:
manifest: Add android_external_jsr330

Change-Id: I507edbb86473cebf2e763a262e03faaa7d13a179

rmcc committed:
Track our own copy of chromium.org's openssl, AGAIN

that was fast... We need new patches here after 4.4.3, so bring back
our own copy

Change-Id: I605c53819feed30ad1e61b3197945788500abb30

jruesga committed:
manifest: Add PhotoPhase repo

Change-Id: Ia2e82ec8fad37cb5ff73a5aa5a19cb16c7b4fc3f
Signed-off-by: Jorge Ruesga <jorge@ruesga.com>

Commit:
Add CMHome application.

Change-Id: I4def8547efbd32a1ee95875b6b18a1459ce4e35d

rmcc committed:
4.4.3_r1.1 -> 4.4.4_r1

Change-Id: I08ab842d171e11974f35ab83e458b15c37dd02bd

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

Commit:
add libraries for dependency injection to external

Change-Id: I852bbd745ba4cf66d7f4d6eabb6157865791299a

ctso committed:
manifest: Remove duplictes, alphabetize

Change-Id: Iac03784a4fad7038bd087346cb2dab9d831c6a8e

Commit:
Add CyanogenMod cardslib fork to default.xml.

Change-Id: I6d767437d33e3d3629af68c725168ed4b11e0ea3

rmcc committed:
4.4.4_r1 -> 4.4.4_r2

Change-Id: I87be9ba59410a2bcc08cc277a40cc4ac0da48a7d

hyperb1iss committed:
Add tinycompress

Change-Id: Ibefccb855c7bb96b236c6f8060340ba807b12756

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

hyperb1iss committed:
Add ANT+ repositories

Change-Id: I3ec0ed5fc1da98d7d40227d385ad8e159ed4b880

ciwrl committed:
Add CMBugreport to manifest

PS2: Fuuu Roman!

Change-Id: I80acbef35206568dbb7f825d76e136bac943125f

Commit:
copy vendor/cm/bootimage/Android.mk to kernel/

Change-Id: I62dd0f89403dc032dd8d6359cee702b7ba58e87e

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

mikeNG committed:
Add missing ANT+ repository

Change-Id: I7258864894fc608f349db716ca8c0acab56c1b39

mikeNG committed:
Add missing ANT+ repository

Change-Id: I7258864894fc608f349db716ca8c0acab56c1b39
(cherry picked from commit 0f1288aef5e519e951d46da6e3aa1b461d9c3b9d)

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

Danesh committed:
Build : Update mockito reference to fork

Change-Id: I4483b3c1c64daedb36f38700d9706f995dc3d359

Commit:
build: reference default branches for mockito

Change-Id: I1c2eea9d017e4c01ba9c797589a9f9c37d3cd7ba

Danesh committed:
Android : Switch back to aosp mockito

Reason : Breaks building of cts

Change-Id: I948363b15075ec58aceceff78ebe4ea72fc9487f

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

jruesga committed:
android: add zip and unrar external repos

Change-Id: Iecdf76bb913f6b70c3106edf4dd20723db485884
Signed-off-by: Jorge Ruesga <jorge@ruesga.com>

cretin45 committed:
Use Mms-caf for Mms

Change-Id: I902f3d3bc552381e3e954dce651a9f43ff4cfd6c

jruesga committed:
android: add libtruezip repo

Change-Id: I92d2b241cfcdaec32136466015d9747b33cd989e
Signed-off-by: Jorge Ruesga <jorge@ruesga.com>

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

Commit:
Use SVOX repository at AOSP due to DMCA takedown

 * Github got a DMCA takedown on this repo based on a licensing dispute with
   a former employee of SVOX. Already have enough drama in my life, just
   point this to Google as we don't make any changes here.

Change-Id: I78f36ed51fd875f5be10486429bc53091bab2860
(cherry picked from commit c61c4e49c043b4da15e122a580b17a96f760c2a9)

Commit:
Merge branch 'cm-11.0' into stable/cm-11.0

Commit:
Deprecate CMAccount

Change-Id: I224f6c0e13f9b9aa380e7bdc3523a753d3cfcf44
Ticket: OSS-59

cm-gerrit commented on README.mkdn:
```diff
@@ -11,7 +11,7 @@ You can do this by using these commands:
     (Go to repo you are patching, make your changes and commit)
```
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit
